python/cuml/test/test_preprocessing.py

```diff
@@ -173,6 +175,19 @@ def test_scale_sparse(sparse_clf_dataset, with_std):  # noqa: F811
     assert_allclose(t_X, sk_t_X)
 
 
+@check_cupy8('pytest')
+@pytest.mark.parametrize("axis", [0, 1])
+def test_maxabs_scaler_funtion(clf_dataset, axis):  # noqa: F811
+    X_np, X = clf_dataset
+
+    t_X = cu_maxabs_scale(X, axis=axis)
+    assert type(t_X) == type(X)
+
+    sk_t_X = sk_maxabs_scale(X_np, axis=axis)
+
+    assert_allclose(t_X, sk_t_X)
```

Comment:
3 little comments here:
1) `check_cupy8` is not necessary anymore and can be safely removed.
2) Can you rename the function `test_maxabs_scale` instead? This is preferable for coherence in naming.
3) During merge, can you add `failure_logger` as the first parameter in this function?